.gitattributes

```diff
@@ -1,4 +1,9 @@
 #
 # set default line endings to always be unix, even on windows systems, for test data
 #
-*.txt text eol=lf
\ No newline at end of file
+*.txt text eol=lf
+
+
+# mark *.proto_builder.go files as auto-generated so they don't clutter a github diff.
+# see https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github
+*.proto_builder.go linguist-generated=true
```

Comment:
Should this be `**/*.proto_builder.go` instead?

Reply:
Wow yes, nice catch